module.json

```diff
@@ -44,7 +44,12 @@
 			"lang": "zh-tw",
 			"name": "正體中文",
 			"path": "lang/zh-tw.json"
-		}
+		},
+        {
```

Comment:
Please use tabs for those indentation as is done in the rest of the file.

Reply:
Fixed, I usually pay attention to indentation, but I missed two lines this time, sorry.